src/app/pages/FrontPage/index.jsx

```diff
@@ -79,6 +84,7 @@ const FrontPage = ({ pageData, mostReadEndpointOverride }) => {
 
   return (
     <>
+      {hasAds && !isAmp && <CanonicalAdBootstrapJs />}
```

Comment:
Would be good to have a comment here explaining why this is not loaded inside the canonical Ad container

Reply:
Added a comment, hopefully it explains enough.